dplace_app/models.py

```diff
@@ -350,6 +347,10 @@ class Language(models.Model):
     # needs to be null=True because some glottolog languages do not have isocodes
     iso_code = models.ForeignKey('ISOCode', null=True)
     family = models.ForeignKey('LanguageFamily', null=True)
+    
+    @property
+    def societies_count(self):
+        return Society.objects.all().filter(language=self).count()
```

Comment:
@xrotwang I did think about this, but wasn't entirely sure how else to do this. Perhaps it would be best to store a count for each language during the data load, like we do for language families (that was my other option, anyway).